src/main/resources/static/common/js/item.js

```diff
@@ -77,31 +77,28 @@ app.controller('itemDetailController', function($rootScope, $http, $location, $f
 	}, 500);
 	
 	//Get cartItem by item id
-	$http({
-		method: 'GET',
-		url: '/proxy/v1/cartItems/' + itemId
-	}).then(function(result) {
-		var cartItem = result.data.data;
-		var quantity = cartItem.quantity;
-		var inventory = cartItem.realInStock;
+    let getCartItemByIdSuccess = (cartItem) => {
+        var quantity = cartItem.quantity;
+        var inventory = cartItem.realInStock;
 
-		if(inventory === 0){$rootScope.itemNum = 0;}
-		$rootScope.itemInventory = inventory;
-		$rootScope.inRequisition = quantity;
-		checkInventory(inventory,quantity,1);
-		$interval(function(){itemDetail.loadingAnimation = false;itemDetail.showQuantity = true;},500,1);
-	}).catch(function(result) {
-		$rootScope.inRequisition = 0;
-		console.info(result);
-		var data = result.data;
-		var status = result.status;
-		if(status === 500){
-			console.log(data.message);
-		}
+        if(inventory === 0) {$rootScope.itemNum = 0;}
+        $rootScope.itemInventory = inventory;
+        $rootScope.inRequisition = quantity;
+        checkInventory(inventory, quantity, 1);
+        $interval(function() {itemDetail.loadingAnimation = false;itemDetail.showQuantity = true;}, 500, 1);
+    }
+    let getCartItemByIdError = (result) => {
```

Comment:
I think there is no need to call `GET /proxy/v1/assets/items/{itemId}` api. Maybe it's a bad implementation when we wrote this code. We can have a discussion at meeting.